common/src/main/kotlin/io/goooler/demoapp/common/util/CommonExtensions.kt

```diff
@@ -165,18 +158,6 @@ inline fun <reified T> DiffUtil.ItemCallback<T>.asConfig(): AsyncDifferConfig<T>
 
 // ---------------------VM & Binding-------------------------------//
 
-@MainThread
-inline fun <reified V, reified VM : BaseThemeViewModel> V.themeViewModels(): Lazy<VM>
-  where V : LifecycleOwner, V : ViewModelStoreOwner, V : ITheme = lazy(LazyThreadSafetyMode.NONE) {
-  ViewModelProvider(this)[VM::class.java].also {
-    lifecycleScope.launch {
-      it.loading.collectLatest { show ->
-        if (show) showLoading() else hideLoading()
-      }
-    }
-  }
-}
-
```

Comment:
Follow up https://github.com/Goooler/DemoApp/pull/75#discussion_r843012980, no need to use `showLoading` & `hideLoading`.